.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+name: "integration-tests"
+on: # rebuild any PRs and main branch changes
+  pull_request:
```

Comment:
Same as above : I'm not sure why we'd need to rebuild on PRs

Reply:
So for this, I expect that we should run tests on each push to know which commit was faulty

I don't really see a point in running tests on PRs